content/200-concepts/100-components/02-prisma-client/056-composite-types.mdx

```diff
@@ -113,7 +113,7 @@ type Photo {
 ...
 ```
 
-The following application code returns the default value if any records have `undefined` in `height` or `width`:
+The following application code returns the default value if any records in the returned document have either `height` or `width` missing:
```

Comment:
Decide if you want to use records or documents, but not both. A record is a document in mongo. I'd leave it at `if any records have either height or width missing on the composite :`

Additionally, the example seems off. `prisma.product.findMany({})` shoudn't fill in `height` or `width`, the entire composite is unset, which doesn't fill in anything. Only if the composite exists somehow, say as an empty embedded document like `photo: {}` on the document _then_ it fills in fields on the composite.

Reply:
I've updated the example to address this point.